internal/settings/settings.go

```diff
@@ -6,7 +6,8 @@ import (
 
 type Options struct {
 	// RootModulePaths describes a list of absolute paths to root modules
-	RootModulePaths []string `mapstructure:"rootModulePaths"`
+	RootModulePaths    []string `mapstructure:"rootModulePaths"`
+	ExcludeModulePaths []string `mapstructure:"excludeModulePaths"`
```

Comment:
I think we should add some validation to reflect that these two options are mutually exclusive. i.e. `RootModulePaths` conflicts with `ExcludeModulePaths`.